### ISSUE TITLE
"Error: EISDIR: illegal operation on a directory" when using -T flag

### ISSUE THREAD

**Issue by simoncoggins** [anvilco/spectaql#280]
I've installed spectaql 1.0.5 on OSX via the command:

```
yarn global add spectaql
```

If I provide a custom theme directory via the config.yml file:
```
% head config.yml
spectaql:
  logoFile: ./logo.svg
  faviconFile: ./favicon.ico
  themeDir: ./themes/spectaql

introspection:
  schemaFile: ./totara.graphqls
  metadataFile: ./metadata.json

```

it works fine and picks up the custom theme but when I try to specify the same theme path via the -T flag I get an error:

```
% spectaql -T ./themes/spectaql config.yml
node:internal/fs/utils:344
    throw err;
    ^

Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (node:fs:723:3)
    at tryReadSync (node:fs:433:20)
    at Object.readFileSync (node:fs:479:19)
    at _default (/usr/local/share/.config/yarn/global/node_modules/spectaql/dist/lib/loadYaml.js:5:35)
    at resolveOptions (/usr/local/share/.config/yarn/global/node_modules/spectaql/dist/index.js:172:56)
    at run (/usr/local/share/.config/yarn/global/node_modules/spectaql/dist/index.js:255:16)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/spectaql/bin/spectaql.js:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47
 {
  errno: -21,
  syscall: 'read',
  code: 'EISDIR'
}
```

**Comment by newhouse** [comment]
Should be fixed in `1.0.8`.

**Comment by simoncoggins** [comment]
Can confirm this is fixed (tested in 1.0.9). Works with -T or --theme-dir and with theme names or a path.
